bosh-cf-crossregion/azuredeploy.json

```diff
@@ -0,0 +1,813 @@
+  "parameters": {
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "name of the vm, will be used as DNS Name for the Public IP used to access the Virtual Machine"
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "User name for the Virtual Machine"
+      }
+    },
+    "sshKeyData": {
+      "type": "string",
+      "metadata": {
+        "description": "Please copy the content of your SSH RSA public key and paste it here. You can use \"ssh-keygen -t rsa -b 2048\" to generate your SSH key pairs."
+      }
+    },
+    "environment": {
+      "type": "string",
+      "allowedValues": [
+        "AzureCloud",
+        "AzureChinaCloud"
+      ],
+      "metadata": {
+        "description": "Different environments in Azure. Choose AzureCloud for Global Azure, and choose AzureChinaCloud for Mooncake (Azure China Cloud)."
+      }
+    },
+    "tenantID": {
+      "type": "string",
+      "minLength": 36,
+      "maxLength": 36,
+      "metadata": {
+        "description": "ID of the tenant. See https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/blob/master/docs/guidance.md"
+      }
+    },
+    "clientID": {
+      "type": "string",
+      "minLength": 36,
+      "maxLength": 36,
+      "metadata": {
+        "description": "ID of the client. See https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/blob/master/docs/guidance.md"
+      }
+    },
+    "clientSecret": {
+      "type": "securestring",
+      "metadata": {
+        "description": "secret of the client. See https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/blob/master/docs/guidance.md"
+      }
+    },
+    "autoDeployBosh": {
+      "type": "string",
+      "defaultValue": "enabled",
+      "allowedValues": [
+        "enabled",
+        "disabled"
+      ],
+      "metadata": {
+        "description": "The flag that enables automatic deployment of Bosh"
+      }
+    },
+    "primaryLocation": {
+      "type": "string",
+      "allowedValues": [
+          "North Europe",
+          "West Europe",
+          "East US",
+          "West US"
+      ],
+      "metadata": {
+        "description": "The primary location to provision resources for Bosh and CF"
+      }
+    },
+    "secondaryLocation": {
+      "type": "string",
+      "allowedValues": [
+          "North Europe",
+          "West Europe",
+          "East US",
+          "West US"
+      ],
+      "metadata": {
+        "description": "The secondary location to provision resources for Bosh and CF"
+      }
+    },
+    "vpnGwSharedKey": {
+      "type": "string",
+      "defaultValue": "youshould2changethisvalueforsure1time",
+      "metadata": {
+        "description": "Shared key used for VPN gateway"
+      }
+    },
+    "systemdomain": {
+      "type": "string",
+      "metadata": {
+        "description": "Domain name used for Cloud Foundry system domain"
+      }
+    },
```

Comment:
@bingosummer `xip.io` will work in this case. But to really enable active/active across two regions, we need to change this value to an actual external domain (for example, www.example.com) that is configured on the Traffic Manager, which will forward traffic to `haproxy` in both regions.